@xen-orchestra/backups/_cleanVm.test.js

```diff
@@ -78,18 +80,18 @@ test('It remove broken vhd', async () => {
   // todo also tests a directory and an alias
 
   await handler.writeFile(`${basePath}/notReallyAVhd.vhd`, 'I AM NOT A VHD')
-  expect((await handler.list(basePath)).length).toEqual(1)
+  assert.strictEqual((await handler.list(basePath)).length, 1)
```

Comment:
`equal` is the same as `strictEqual` when using `assert.strict`.

Same goes for `deepEqual` and `deepStrictEqual`.

Reply:
Ok, I'll change that